### PR TITLE
[WWST-2361] Added support for Aeotec NanoMote Quad

### PR DESCRIPTION
**Pull request by PKacprowiczS**
@greens @MMateusiakS @MGoralczykS

### CODE REVIEW

**Comment by greens** [comment]
generic-4-button doesn't support double-pressed. Do the nanomotes?

**Comment by PKacprowiczS** [comment]
Nope, they only support pressed and held. I've left double because it is supported by Fibaro devices and it's already working part of Button Capability.

File: devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy [hunk 1]
```diff
@@ -54,7 +58,11 @@ def updated() {
 def initialize() {
 	def numberOfButtons = prodNumberOfButtons[zwaveInfo.prod]
 	sendEvent(name: "numberOfButtons", value: numberOfButtons, displayed: false)
-	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	if(zwaveInfo.mfr?.contains("0371")) {
```
Comment (kianooshkarami) [inline devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy]: Can we have a helper function that defines which manufacturer this is?

File: devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy [hunk 2]
```diff
@@ -54,7 +58,11 @@ def updated() {
...
+		sendEvent(name: "DeviceWatch-Enroll", value: JsonOutput.toJson([protocol: "zwave", scheme:"untracked"]), displayed: false)
+	} else {
+		sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```
Comment (kianooshkarami) [inline devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy]: Is the wake up interval 4 hours? 

Sorry for repeating myself across multiple PRs, but the check interval needs to be 
```

func calculate_check_interval(w: wakeup_interval)
{
    check_interval = 2 * w + 10 minutes;
}
```

The 10 minutes should be the offset used going forward, the reason being that if we miss the first wake up interval (at the radio level), the device could stay up for 5 minutes before going back to sleep and wake up 4 hours or so later. Also most of these low power devices are running on Low power internal clocks which can be inaccurate, so the 4 hours wake up interval could sleep by seconds or more, so the 10 minutes should be a decent cushion that when elapsed we can feel safe to mark the device offline

File: devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy [hunk 3]
```diff
@@ -74,8 +82,11 @@ def initialize() {
 
 def configure() {
 	def cmds = []
-	for(def parameter : 21..26) {
-		cmds += secure(zwave.configurationV1.configurationSet(parameterNumber: parameter, scaledConfigurationValue: 15)) //Makes Fibaro KeyFob buttons send all kind of supported events
+	if(zwaveInfo.mfr?.contains("010F")) {
```
Comment (kianooshkarami) [inline devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy]: Please use a helper function to identify the manufacturer

File: devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy (hunk 2 above)
Comment (PKacprowiczS) [reply devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy]: Sorry, totally forgot about it. Now it should be ok.

**Review by greens — APPROVED** [review]

**Pull request merged**